docs/reference/operations-guide/federated-operation.md

```diff
@@ -0,0 +1,730 @@
+    ```json
+    {
+        "class": "uk.gov.gchq.gaffer.operation.OperationChain",
+        "operations": [
+            {
+                "class": "uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation",
+                "operation": {
+                    "class": "ExampleOperation1"
+                },
+                "graphIds": [ "graphA" ]
+            },
+            {
+                "class": "uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation",
+                "operation": {
+                    "class": "ExampleOperation2"
+                },
+                "graphIds": [ "graphB" ]
+            }
+        ]
+    }
+    ```
+??? example "Sending a single operation to one subgraph and a chain of operations to another subgraph in your Federated Store"
+    ```json
+    {
+        "class": "uk.gov.gchq.gaffer.operation.OperationChain",
+        "operations": [
+            {
+                "class": "uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation",
+                "operation": {
+                    "class": "uk.gov.gchq.gaffer.operation.OperationChain",
+                    "operations": {
+                        [
+                            "class": "ExampleOperation1",
+                            "class": "ExampleOperation2"
+                        ]
+                    }
+                },
+                "graphIds": [ "graphA" ]
+            },
+            {
+                "class": "uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation",
+                "operation": {
+                    "class": "ExampleOperation3"
+                },
+                "graphIds": [ "graphB" ]
+            }
+        ]
+    }
+    ```
+
+As you can see in the example above we have wrapped an OpertationChain inside a
+Federated Operation.
+
+### The Merge Function
+
+Merge functions  dictate how the FederatedStore will merge results from
+different subgraphs dependent on the Operation.
+
+The examples below refer to this graph.
+
+```mermaid
+graph LR
+    subgraph Federated Store
+        subgraph Graph A
+            direction TB
+            A((Person<br>id: 1<br>name: bob)) --works as--> B((Job<br>id: 2<br>title: builder))
+            A --lives in--> C((Place<br>id: 3<br>name: bobsville))
+        end
+            subgraph Graph B
+            direction TB
+            D((Person<br>id: 1<br>name: bob)) --drives--> E((Vehicle<br>id: 4<br>type: digger))
+            E --make/model--> F((Manufacturer<br>id: 5<br>name: JCB))
+        end
+    end
+```
+
+#### ApplyViewToElementsFunction
+
+This merge function uses the View from the operation and applies it to all of
+the results, meaning the results are now re-aggregated and re-filtered using the
+Schema, locally in the FederatedStore. This makes the results look like they
+came from one graph, rather than getting back an array of Elements from the
+different subgraphs.  
+
+
+??? Example "Example using the GetAllElements operation"
```

Comment:
Is it worth adding a simple `View` to this example to demo how the merge function would apply it to the merged results from the two graphs e.g. like 

```json
"view": {
    "entities": {
        "Person": {
            "properties": [
                "Bob"
            ]
        }
    }
}
```

Reply:
Added in the count property to the Person entity which will display this merging feature.